src/assets/locales/fr.json

```diff
@@ -0,0 +1,567 @@
+{
+  "auto-exit-warning-modal.message": "Fin de session automatique dans {secondsRemaining} secondes",
+  "auto-exit-warning-modal.reason.concurrent-session": "Vous avez commencé une autre session.",
+  "auto-exit-warning-modal.reason.idle": "Vous êtes resté(e) en pause trop longtemps.",
+  "avatar-editor.advanced-section": "Avancé",
+  "avatar-editor.custom-avatar-option": "GLB personnalisé...",
+  "avatar-editor.delete-avatar-button": "Supprimer l'Avatar",
+  "avatar-editor.delete-avatar.cancel": "non",
+  "avatar-editor.delete-avatar.confirm": "oui",
+  "avatar-editor.delete-avatar.confirmation-prompt": "Etes vous sûr(e)?",
```

Comment:
I tried "Confirmer ?" in the old UI translation to avoid gender declinaison
Apply to all?